WordPress/Classes/ViewRelated/Site Creation/Site Segments/SiteSegmentsStep.swift

```diff
@@ -8,7 +8,7 @@ final class SiteSegmentsStep: WizardStep {
         return SiteSegmentsWizardContent(service: self.service, selection: self.didSelect)
     }()
 
-    var delegate: WizardDelegate?
+    weak var delegate: WizardDelegate?
```

Comment:
A retain cycle between `WizardNavigation` and `SiteSegmentsStep`

https://github.com/wordpress-mobile/WordPress-iOS/blob/48b8a391394f2b4aee9b886abe21f62a521c0a81/WordPress/Classes/ViewRelated/Site%20Creation/Wizard/WizardNavigation.swift#L47-L51